modules/features/manatime_feature_timesheets/themes/timesheet-view.tpl.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Template file for timesheet.
+ */
+ ?>
+
+ <main id="time-entries-main">
+   <?php if (empty($records)): ?>
+     <p>empty</p>
+   <?php else: ?>
+       <table class="time-time_entries-list">
+         <tr>
+           <th>Description</th>
+           <th>Sunday</th>
```

Comment:
Try to get this strings from php in order to make them translatable. Or Print them here with t <?php print t('Sunday'); ?>